src/sentry/utils/sdk.py

```diff
@@ -64,6 +64,7 @@
     "sentry.tasks.store.process_event": settings.SENTRY_PROCESS_EVENT_APM_SAMPLING,
     "sentry.tasks.store.process_event_from_reprocessing": settings.SENTRY_PROCESS_EVENT_APM_SAMPLING,
     "sentry.tasks.assemble.assemble_dif": 0.1,
+    "sentry.tasks.process_suspect_commits": 0.1,
```

Comment:
Would this mean enabling tracing also for onpremise?

Reply:
fixed to use a setting